src/traces/scatterpolargl/index.js

```diff
@@ -26,22 +27,28 @@ function calc(container, trace) {
     var angularAxis = layout[subplotId].angularaxis;
     var rArray = radialAxis.makeCalcdata(trace, 'r');
     var thetaArray = angularAxis.makeCalcdata(trace, 'theta');
+    var len = trace._length;
     var stash = {};
 
-    if(trace._length < rArray.length) rArray = rArray.slice(0, trace._length);
-    if(trace._length < thetaArray.length) thetaArray = thetaArray.slice(0, trace._length);
-
-    calcColorscales(trace);
+    if(len < rArray.length) rArray = rArray.slice(0, len);
+    if(len < thetaArray.length) thetaArray = thetaArray.slice(0, len);
 
     stash.r = rArray;
     stash.theta = thetaArray;
 
-    trace._extremes.x = Axes.findExtremes(radialAxis, rArray, {tozero: true});
+    // We could add TOO_MANY_POINTS logic like in Scattergl.calc,
+    // if users asks for scaterpolargl charts with > 1e5 pts
```

Comment:
isn't that more or less the purpose of `scatterpolargl`? I'd vote to just add it now.

Reply:
done in https://github.com/plotly/plotly.js/pull/3098/commits/25fb7dc583b0d2e2b424d23add79a18636f35b9f